### PR TITLE
Adds init script to bootstrap setup

### PR DESCRIPTION
**Pull request by weaverryan**
This is a simpler alternative to #307.

In short, I want to make bootstrapping Encore easier and more consistent. This adds an `init` script to Encore, which asks *just* a few questions, and bootstraps a simple `webpack.config.js` file and one very simple entry. Here's what it looks like as a screenshot:
screen shot 2018-05-08 at 3 34 36 pm
screen shot 2018-05-08 at 3 34 47 pm

It generates a very simple file structure, which is certainly open for debate:

```
assets/
    js/
        layout.js
    css/
        layout.css
```

In other words, it "pushes" the user towards a file structure similar to the Symfony demo: https://github.com/symfony/demo/tree/master/assets

TODOs:

* [ ] make it work with npm
* [ ] make the css/ directory dynamic - i.e. could be less or scss instead
* [ ] get tests passing
* [ ] update .gitignore for `/node_modules/
* [ ] maybe re-add code to add "scripts" to package.json

### CODE REVIEW

**Comment by stof** [comment]
@weaverryan does it make sense to keep this `init-generator` branch in the repo to finish the work, or is it abandoned ?
